### PR TITLE
Add project templates

### LINKED ISSUE #2134: Show first templates in dashboard
To let users start from a template instead of a blank canvas, we need to add some starter templates now.

## Todo
- [x] Hardcoded list of projects that can be cloned
- [x] Hardcoded permissions for that list to allow cloning for everyone
- [x] Section "Templates" under Projects
- [x] Items are the same but no menu
- [x] Click should open Dialog (same as rename dialog)
- [x] Click "Create" creates a clone with that name and opens in builder

### PR DESCRIPTION
**Pull request by istarkov**
## Description

closes #2134

image

On card click

image

Also fixed resize

image

Made it

image

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/env/env.server.ts [hunk 1]
```diff
@@ -77,6 +77,14 @@ const env = {
...
   ENTRI_APPLICATION_ID: process.env.ENTRI_APPLICATION_ID ?? "webstudio",
   ENTRI_SECRET: process.env.ENTRI_SECRET,
+
+  /**
+   * Projects as templates in dashboard
+   */
+  PROJECT_TEMPLATES:
+    process.env.PROJECT_TEMPLATES?.split(",").map((projectId) =>
+      projectId.trim()
+    ) ?? [],
```
Comment (TrySound) [inline apps/builder/app/env/env.server.ts:87]: .split(/\s*,\s*/) ?? []

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/dashboard/dashboard.stories.tsx [hunk 2]
```diff
@@ -26,7 +26,9 @@ const createRouter = (element: JSX.Element) =>
   ]);
 
 export const Empty: ComponentStory<typeof Dashboard> = () => {
-  const router = createRouter(<Dashboard user={user} projects={[]} />);
+  const router = createRouter(
+    <Dashboard user={user} projects={[]} projectTemplates={[]} />
```
Comment (kof) [inline apps/builder/app/dashboard/dashboard.stories.tsx:30]: should we call them "clonables"?

Comment (kof) [reply apps/builder/app/dashboard/dashboard.stories.tsx:30]: sorry, ignore it, templates is correct

**Review by kof — APPROVED** [review]
awesome!

File: apps/builder/app/env/env.server.ts (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/env/env.server.ts:87]: Will not solve the edgecases when 1st or last item has spaces at begin and end.

**Pull request merged**